mne/viz/evoked.py

```diff
@@ -1191,3 +1191,342 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     # show and return it
     plt_show(show)
     return fig
+
+
+def _ci(arr, ci):
+    """Calculate the `ci`% parametric confidence interval for `arr`"""
+    from scipy import stats
+    mean, sigma = arr.mean(0), stats.sem(arr, 0)
+    # This is highly convoluted to support 17th century Scipy
+    # XXX Fix when Python 2.6 support is dropped!
+    return np.asarray([stats.t.interval(ci, arr.shape[0],
+                       loc=mean_, scale=sigma_)
+                       for mean_, sigma_ in zip(mean, sigma)]).T
+
+
+def plot_compare_evokeds(evokeds, picks=None, conditions=None, ch_names=None,
+                         colors=None, linestyles=['-'], styles=None,
+                         vlines=[0], ci=0.95, truncate_yaxis=True, ymin=None,
+                         ymax=None, invert_y=False, ax=None, title=None,
+                         show=True):
+    """ Plot, for multiple conditions, individual evoked time courses of one or
+    the mean of multiple sensors
+
+    This function is useful for comparing ER[P/F]s at a specific location. It
+    plots grand averages or, if supplied with a list/dict of lists of evoked
+    instances, grand averages plus confidence intervals.
+
+    Parameters
+    ----------
+    evokeds : instance of mne.Evoked | list | dict
```

Comment:
you do not mention the case when `evokeds` is a list of Evoked's (condition labels are then just `'1', '2', ...`, right?)